### PR TITLE
Remove small social icons

### PR DESCRIPTION
**Pull request by joshcanhelp**
### Changes

- Remove small social button option
- Fix Embedded setting validation

### References

[Lock removing small social button config option](https://github.com/auth0/lock/pull/1637)

### Testing

* [x] This change adds unit test coverage
* [x] This change has been tested on WP 5.1.1

### Checklist

* [x] All existing and new tests complete without errors
* [x] All code quality tools/guidelines in the [Contribution guide](CONTRIBUTION.md) have been run/followed
* [x] All active GitHub CI checks have passed

### CODE REVIEW

File: WP_Auth0.php [hunk 1]
```diff
@@ -9,8 +9,8 @@
  * Text Domain: wp-auth0
  */
 
-define( 'WPA0_VERSION', '3.10.0' );
-define( 'AUTH0_DB_VERSION', 21 );
+define( 'WPA0_VERSION', '3.11.0-beta' );
```
Comment (joshcanhelp) [inline WP_Auth0.php]: Set a plugin version that can be updated when the GA version is released

File: WP_Auth0.php [hunk 2]
```diff
@@ -9,8 +9,8 @@
...
+define( 'AUTH0_DB_VERSION', 22 );
```
Comment (joshcanhelp) [inline WP_Auth0.php]: Trigger DB update

File: lib/WP_Auth0_Lock10_Options.php [hunk 3]
```diff
@@ -111,9 +111,8 @@ protected function build_settings( $settings ) {
 
 		}
 
-		if ( $this->_is_valid( $settings, 'social_big_buttons' ) ) {
-			$options_obj['socialButtonStyle'] = $settings['social_big_buttons'] ? 'big' : 'small';
-		}
+		$options_obj['socialButtonStyle'] = 'big';
```
Comment (joshcanhelp) [inline lib/WP_Auth0_Lock10_Options.php]: Always set to `big`

File: lib/admin/WP_Auth0_Admin_Appearance.php [hunk 4]
```diff
@@ -472,19 +468,18 @@ public function render_custom_signup_fields( $args = array() ) {
...
 	public function basic_validation( $old_options, $input ) {
-		$input['form_title']         = sanitize_text_field( $input['form_title'] );
-		$input['icon_url']           = esc_url( $input['icon_url'], array( 'http', 'https' ) );
-		$input['social_big_buttons'] = ( isset( $input['social_big_buttons'] ) ? $input['social_big_buttons'] : 0 );
-		$input['gravatar']           = ( isset( $input['gravatar'] ) ? $input['gravatar'] : 0 );
-		$input['language']           = sanitize_text_field( $input['language'] );
-		$input['primary_color']      = sanitize_text_field( $input['primary_color'] );
+		$input['form_title']    = empty( $input['form_title'] ) ? '' : sanitize_text_field( $input['form_title'] );
+		$input['icon_url']      = empty( $input['icon_url'] ) ? '' : esc_url( $input['icon_url'], array( 'http', 'https' ) );
+		$input['gravatar']      = isset( $input['gravatar'] ) ? $input['gravatar'] : 0;
```
Comment (damieng) [inline lib/admin/WP_Auth0_Admin_Appearance.php]: I know you didn't change the logic around here but should the gravatar have a sanitize or escape?

Comment (joshcanhelp) [reply lib/admin/WP_Auth0_Admin_Appearance.php]: It's checked as truth-y. Any value == `true`

Comment (joshcanhelp) [reply lib/admin/WP_Auth0_Admin_Appearance.php]: More specifically ... any value is stored, then a truth-y value means "use the incoming Auth0 avatar"

**Pull request merged**